examples/lapack_like/HermitianEig.cpp

```diff
@@ -7,12 +7,88 @@
    http://opensource.org/licenses/BSD-2-Clause
 */
 #include <El.hpp>
+
 using namespace std;
 using namespace El;
 
-// Typedef our real and complex types to 'Real' and 'C' for convenience
-typedef double Real;
-typedef Complex<Real> C;
+template<typename Real>
+void run_example(Int n, bool print)
```

Comment:
Thanks for the comment, I'm going to fix that.

By being able to disable each precision you mean a compilation flag? Or runtime argument?